src/yui/js/yui.js

```diff
@@ -527,7 +527,7 @@ proto = {
         // using eval(). This is critical for Content Security Policy enabled
         // sites and other environments like Chrome extensions
         if (!Y.config.hasOwnProperty('global')) {
-            Y.config.global = Function('return this')();
+            Y.config.global = window || global;
```

Comment:
would `globalThis` work? we're using a super-old node version which doesn't understand `globalThis` but at the end of the day this should never hit that runtime but a modern browser

Reply:
I put `global` just to be safe in case it is needed in any test or something like that but, TBH, I don't think we need anything different from `window` since the code is designed solely to be run in browsers :shrug: .

Reply:
BTW, I didn't know about `globalThis` but it seems it's ES2020 -> https://www.javascripttutorial.net/es-next/javascript-globalthis/ so I think it's better to use `window` for now since it will be better supported by browsers...